### ISSUE TITLE
Values hover close to mid-scale

### ISSUE THREAD

**Issue by djrapp** [dwpicott/Emotion-Prediction-CNN#1]
Hello,

I was excited to try out your code with my Muse 2. I am observing that the values for all 3 axes tend to stay very close to 5 (i.e., right around the origin). I cannot seem to make the dot travel very far from the origin at all. Have you encountered this behavior as well? Thank you!

**Comment by dwpicott** [comment]
Yes, that's the usual behaviour. This was created more as a proof-of-concept than a practical application. The model isn't very accurate and will tend towards the mean value when it is uncertain. There are certainly improvements that could be made to the training, noise filtering, etc. but I'm not currently planning to continue work on it.
